### ISSUE TITLE
🐛 Can't support the purelib setup of Tensorflow 2.0.0

### ISSUE THREAD

**Issue by thundergolfer** [dillon-giacoppo/rules_python_external#8]
#### Steps to reproduce 

Setup `rules_python_external` with:

```
# requirements.txt
tensorflow==2.0.0
```

Do `bazel sync`.

#### What was expected 

Sync would work fine and the Tensorflow package would be available to use with `requirement`. 

### What happened: 

```
INFO: Call stack for the definition of repository 'pypi' which is a pip_repository (rule definition at /private/var/tmp/_bazel_jbelotti/a8a1f81a1422a72c79c6c81641a8c816/external/rules_python_external/defs.bzl:25:18):
 - /Users/jbelotti/work/data-science/WORKSPACE:100:1
ERROR: An error occurred during the fetch of repository 'pypi':
   failed to create pip repository:  (Traceback (most recent call last):
  File "/private/var/tmp/_bazel_jbelotti/a8a1f81a1422a72c79c6c81641a8c816/external/rules_python_external/tools/wheel_wrapper.py", line 67, in <module>
    main()
  File "/private/var/tmp/_bazel_jbelotti/a8a1f81a1422a72c79c6c81641a8c816/external/rules_python_external/tools/wheel_wrapper.py", line 63, in main
    whl.main()
  File "/private/var/tmp/_bazel_jbelotti/a8a1f81a1422a72c79c6c81641a8c816/external/rules_python_external/./src/extract_wheels.py", line 127, in main
    extract_wheel(wheel, whl_label, [])
  File "/private/var/tmp/_bazel_jbelotti/a8a1f81a1422a72c79c6c81641a8c816/external/rules_python_external/./src/extract_wheels.py", line 83, in extract_wheel
    purelib.spread_purelib_into_root(directory)
  File "/private/var/tmp/_bazel_jbelotti/a8a1f81a1422a72c79c6c81641a8c816/external/rules_python_external/./src/purelib.py", line 32, in spread_purelib_into_root
    dst=extracted_whl_directory,
  File "/nix/store/i9zg3n6dgi7jdw37nxyfi272pszkk6nd-python3-3.6.9/lib/python3.6/shutil.py", line 548, in move
    raise Error("Destination path '%s' already exists" % real_dst)
shutil.Error: Destination path 'pypi__tensorflow/tensorflow_core' already exists
)
ERROR: no such package '@pypi//': failed to create pip repository:  (Traceback (most recent call last):
  File "/private/var/tmp/_bazel_jbelotti/a8a1f81a1422a72c79c6c81641a8c816/external/rules_python_external/tools/wheel_wrapper.py", line 67, in <module>
    main()
  File "/private/var/tmp/_bazel_jbelotti/a8a1f81a1422a72c79c6c81641a8c816/external/rules_python_external/tools/wheel_wrapper.py", line 63, in main
    whl.main()
  File "/private/var/tmp/_bazel_jbelotti/a8a1f81a1422a72c79c6c81641a8c816/external/rules_python_external/./src/extract_wheels.py", line 127, in main
    extract_wheel(wheel, whl_label, [])
  File "/private/var/tmp/_bazel_jbelotti/a8a1f81a1422a72c79c6c81641a8c816/external/rules_python_external/./src/extract_wheels.py", line 83, in extract_wheel
    purelib.spread_purelib_into_root(directory)
  File "/private/var/tmp/_bazel_jbelotti/a8a1f81a1422a72c79c6c81641a8c816/external/rules_python_external/./src/purelib.py", line 32, in spread_purelib_into_root
    dst=extracted_whl_directory,
  File "/nix/store/i9zg3n6dgi7jdw37nxyfi272pszkk6nd-python3-3.6.9/lib/python3.6/shutil.py", line 548, in move
    raise Error("Destination path '%s' already exists" % real_dst)
shutil.Error: Destination path 'pypi__tensorflow/tensorflow_core' already exists
)
```

**Comment by thundergolfer** [comment]
I've pulled down https://files.pythonhosted.org/packages/46/0f/7bd55361168bb32796b360ad15a25de6966c9c1beb58a8e30c01c8279862/tensorflow-2.0.0-cp36-cp36m-manylinux2010_x86_64.whl and had a look at it. 

It has a filetree where part of it is: 

```
tensorflow_core/
tensorflow-2.0.0.data/tensorflow_core/
```

So that's why it's breaking. I don't understand wheels enough to know why they'd both exists and what to do when they do, but I'll figure it out. 

I think Tensorflow 2.0.0 was working prior to https://github.com/dillon-giacoppo/rules_python_external/pull/6 so maybe it's a case of doing nothing in this situation...? 

cc @dillon-giacoppo heads up about this. I'll fix it 👍

**Comment by thundergolfer** [comment]
Fixed by #9
